kolibri/plugins/coach/assets/src/modules/lessonResources/handlers.js

```diff
@@ -267,18 +267,22 @@ function _prepLessonContentPreview(store, classId, lessonId, contentId) {
     getParams: { no_available_filtering: true },
   }).then(
     contentNode => {
-      const assessmentMetadata = contentNode.assessmentmetadata;
       store.commit('lessonSummary/SET_STATE', {
         toolbarRoute: {},
         // only exist if exercises
         workingResources: null,
         resourceCache: cache,
       });
+
```

Comment:
In case this looks mistaken I did add it on purpose to put visual space between the logical blocks a bit